app/services/search/query_builders/feed_content.rb

```diff
@@ -148,6 +152,8 @@ def term_keys
         end.compact
       end
 
+      # Search fields based on a range
```

Comment:
Here we can leverage Postgres itself, as FTS queries can be combined with other conditions.